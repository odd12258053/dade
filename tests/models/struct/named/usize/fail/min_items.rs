use dade::model;
#[model]
struct TestModel {
    #[field(min_items = 2)]
    value: usize,
}
fn main() {}
