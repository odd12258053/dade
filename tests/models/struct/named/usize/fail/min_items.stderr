error: Support condition is gt, ge, lt, le, alias, default and validate
 --> tests/models/struct/named/usize/fail/min_items.rs:5:5
  |
5 |     value: usize,
  |     ^^^^^
